src/js/tabs/firmware_flasher.js

```diff
@@ -118,21 +157,52 @@ TABS.firmware_flasher.initialize = function (callback) {
                         });
                     });
                 TABS.firmware_flasher.releases = releases;
+
                 chrome.storage.local.get('selected_board', function (result) {
                     if (result.selected_board) {
-                        $('select[name="board"]').val(result.selected_board);
-                        $('select[name="board"]').trigger("change");
+                        var boardReleases = releases[result.selected_board]
+                        $('select[name="board"]').val(boardReleases ? result.selected_board : 0).trigger('change');
                     }
                 });
             }
         };
 
+        var buildType_e = $('select[name="build_type"]')
+            .append($("<option value='development'>Development</option>"))
```

Comment:
We should probably only show the `Development` option when expert mode is enabled.

Reply:
And here the options must be translated too (development and the akk patch).